AzureCalling/Controllers/CallViewController.swift

```diff
@@ -629,6 +631,21 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
             bottomDrawerViewController.refreshBottomDrawer()
         }
     }
+    // update the audio device icon with current selection
+    private func updateAudioDevice() {
```

Comment:
naming too generic it's not actually updating the audio device, should probably include update button icon in the name